<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CInterpolationSetupComponent</class>
 <widget class="QFrame" name="CInterpolationSetupComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interpolation setup</string>
  </property>
  <layout class="QVBoxLayout" name="vl_InterpolationSetup">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="BlackGui::Views::CInterpolationSetupView" name="tvp_InterpolationSetup">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="fr_Mode">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="hl_Mode">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="BlackGui::Components::CCallsignCompleter" name="comp_CallsignCompleter">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_Callsign">
        <property name="text">
         <string>callsign</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_Global">
        <property name="text">
         <string>global</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_Reload">
        <property name="text">
         <string>reload</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="BlackGui::Editors::CInterpolationSetupForm" name="form_InterpolationSetup">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="fr_Buttons">
     <layout class="QHBoxLayout" name="hl_Buttons">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="cb_IgnoreGlobal">
        <property name="toolTip">
         <string>no setups same as global setup</string>
        </property>
        <property name="text">
         <string>â‰ global</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hs_Buttons">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pb_RenderingSetup">
        <property name="toolTip">
         <string>goto rendering setup</string>
        </property>
        <property name="text">
         <string> rendering</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_Delete">
        <property name="text">
         <string>delete</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_Save">
        <property name="text">
         <string>save</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BlackGui::Components::CCallsignCompleter</class>
   <extends>QFrame</extends>
   <header>blackgui/components/callsigncompleter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Views::CInterpolationSetupView</class>
   <extends>QTableView</extends>
   <header>blackgui/views/interpolationsetupview.h</header>
  </customwidget>
  <customwidget>
   <class>BlackGui::Editors::CInterpolationSetupForm</class>
   <extends>QFrame</extends>
   <header>blackgui/editors/interpolationsetupform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tvp_InterpolationSetup</tabstop>
  <tabstop>rb_Callsign</tabstop>
  <tabstop>rb_Global</tabstop>
  <tabstop>pb_Reload</tabstop>
  <tabstop>cb_IgnoreGlobal</tabstop>
  <tabstop>pb_RenderingSetup</tabstop>
  <tabstop>pb_Delete</tabstop>
  <tabstop>pb_Save</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
